src/api/admin.ts

```diff
@@ -295,34 +295,23 @@ const createFirestoreRoomInput = async (
   const urlPortalName = createSlug(
     input.title + Math.random().toString() //room titles are not necessarily unique
   );
-  type ImageNaming = {
-    fileKey: RoomImageFileKeys;
-    urlKey: RoomImageUrlKeys;
-  };
-  const imageKeys: Array<ImageNaming> = [
-    {
-      fileKey: "image_file",
-      urlKey: "image_url",
-    },
-  ];
 
   let imageInputData = {};
 
   // upload the files
-  for (const entry of imageKeys) {
-    const fileArr = input[entry.fileKey];
+  if (input["image_file"]) {
+    const fileArr = input["image_file"];
     const downloadUrl = await uploadFile(
       `users/${user.uid}/venues/${venueId}/${urlPortalName}`,
       fileArr
     );
-    imageInputData = { ...imageInputData, [entry.urlKey]: downloadUrl };
+    imageInputData = { image_url: downloadUrl };
```

Comment:
@goran-peoski-work I don't think that we've used to pass `image_file` there, as `[entry.urlKey]: downloadUrl ` equals to `"image_url": url` and `imageInputData` shouldn't include that `image_file` at all

Reply:
`image_file` is only set if someone has attached an image via a form.
`image_url` is the URL that was passed to the form from the database.
If `image_file` is set then we should use that (and `image_url` gets set by the cloud function`). Otherwise, just pass `image_url` to the cloud function.